spec/std/dir_spec.cr

```diff
@@ -355,19 +382,19 @@ describe "Dir" do
 
   describe ".tempdir" do
     it "returns default directory for tempfiles" do
-      old_tmpdir = ENV["TMPDIR"]?
-      ENV.delete("TMPDIR")
-      Dir.tempdir.should eq("/tmp")
-    ensure
-      ENV["TMPDIR"] = old_tmpdir
+      unset_tempdir do
+        # FIXME: [win32] GetTempPathW returns the Windows directory, which is not
+        # necessarily `C:\\WINDOWS`.
+        Dir.tempdir.should eq {{ flag?(:windows) ? "C:\\WINDOWS" : "/tmp" }}
```

Comment:
But what if it's `C:\windows`? :grimacing: 

I'm eager to get this fixed because for me it's [the last failing spec](https://github.com/oprypin/crystal/runs/384714527#step:15:45).

If there's no easy solution, we could just put `pending_win32` on this one.

Reply:
We could simply call `upcase`, windows paths are usually case-insensitive :laughing:

Reply:
Oh btw, this spec is actually failing on Mac in this state (see circleci)

Reply:
Technically, we could verify the value using `GetWindowsDirectoryW`. But I don't want to go that deep into the implementation details of the OS API.

It's probably best to simply verify that the directory exists which should be the case for Windows directory.